editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -4561,6 +4567,15 @@ Node3DEditorViewport::Node3DEditorViewport(Node3DEditor *p_spatial_editor, Edito
 		view_menu->get_popup()->set_item_tooltip(shadeless_idx, unsupported_tooltip);
 	}
 
+	register_shortcut_action("spatial_editor/freelook_left", TTR("Freelook Left"), Key::A);
+	register_shortcut_action("spatial_editor/freelook_right", TTR("Freelook Right"), Key::D);
+	register_shortcut_action("spatial_editor/freelook_forward", TTR("Freelook Forward"), Key::W);
+	register_shortcut_action("spatial_editor/freelook_backwards", TTR("Freelook Backwards"), Key::S);
+	register_shortcut_action("spatial_editor/freelook_up", TTR("Freelook Up"), Key::E);
+	register_shortcut_action("spatial_editor/freelook_down", TTR("Freelook Down"), Key::Q);
+	register_shortcut_action("spatial_editor/freelook_speed_modifier", TTR("Freelook Speed Modifier"), Key::SHIFT);
+	register_shortcut_action("spatial_editor/freelook_slow_modifier", TTR("Freelook Slow Modifier"), Key::ALT);
+
 	ED_SHORTCUT("spatial_editor/freelook_left", TTR("Freelook Left"), Key::A);
 	ED_SHORTCUT("spatial_editor/freelook_right", TTR("Freelook Right"), Key::D);
 	ED_SHORTCUT("spatial_editor/freelook_forward", TTR("Freelook Forward"), Key::W);
```

Comment:
Aren't those redundant now? `register_shortcut_action` seems to do that + more.

Reply:
Hmm yep, looks like that could have been a re basing mistake.